functiondefextractor/core_extractor.py

```diff
@@ -1,11 +1,18 @@
 """This file contains all the functions required for extracting method/function definitions from the given repository"""
+import datetime
 import subprocess
 import os
 import re
+import sys
+import time
+
 import pandas as pd
+import extractor_log as cl
 
+LOG = cl.get_logger()
 DELTA_BODY = []
 UID_LIST = []
+FILE_TYPE = [".java", ".cpp", ".c", ".cs", ".py", ".ts", ".js"]  # pragma: no mutate
```

Comment:
That is a long file :) . Would you like to consider splitting the file based on functionality ?
Would like to attempt in OO way ?

Reply:
this initial step concentrated on requirements. In future versions OO way structure will be introduced